packages/trader/src/Modules/SmartChart/Components/Markers/accumulators-profit-loss-text.tsx

```diff
@@ -1,10 +1,25 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import { Text } from '@deriv/components';
 import { FastMarker } from 'Modules/SmartChart';
 import classNames from 'classnames';
+import AccumulatorsProfitLossTooltip from './accumulators-profit-loss-tooltip';
 
-const ACTIONS = {
+type TAccumulatorsProfitLossText = Omit<
+    React.ComponentProps<typeof AccumulatorsProfitLossTooltip>,
+    'alignment' | 'exit_tick' | 'exit_tick_time' | 'high_barrier' | 'is_sold'
+>;
+
+type TRef = {
+    setPosition: (position: { epoch: number | null; price: number | null }) => void;
+};
+
+type TActions = {
+    INC: string;
+    DEC: string;
+    ADD10: string;
+};
```

Comment:
suggest having a ts props file in the main folder and move the TS types there.

Reply:
They are local to the component. I think they should stay in the component itself. I did export and reused TRef since its shared in two components.